internal/config/state.go

```diff
@@ -218,3 +220,47 @@ func isValidImage(image string) bool {
 
 	return ref != "" && tag != ""
 }
+
+// ReadResponse reads the library state from configure-response.json.
+//
+// The response file is removed afterwards.
+func ReadResponse(contentLoader func(data []byte, state *LibraryState) error, jsonFilePath string) (*LibraryState, error) {
```

Comment:
FYI, you can omit "data" and "state" in the func on this line to make it shorter. But it's also OK to leave them there.